src/initializers/riviere.ts

```diff
@@ -11,6 +11,25 @@ const https = require('https');
 let middleware;
 
 const init = (config, orkaOptions) => {
+  // TODO: Remove this conversion logic after v5.x.x release
+  const logger = getLogger('orka.riviere');
+  const regexKeys = ['headersRegex', 'bodyKeysRegex'];
+  regexKeys.forEach(key => {
+    if (typeof config?.riviere[key] === 'string') {
+      logger.warn(
+        `You are using a string for regex key ${key} in riviere config. This will not supported after Orka v5.x.x. Please use a RegExp object.`
```

Comment:
```suggestion
        `You are using a string for regex key ${key} in riviere config. This will not be supported after Orka v5.x.x. Please use a RegExp object.`
```